packages/expo-image/android/src/main/java/expo/modules/image/enums/ImageResizeMode.java

```diff
@@ -0,0 +1,37 @@
+package expo.modules.image.enums;
+
+import android.widget.ImageView.ScaleType;
+
+public enum ImageResizeMode {
+    UNKNOWN("unknown", null),
+    CONTAIN("contain", ScaleType.FIT_CENTER),
+    COVER("cover", ScaleType.CENTER_CROP),
+    STRETCH("stretch", ScaleType.FIT_XY),
+    CENTER("center", ScaleType.CENTER),
+    REPEAT("repeat", ScaleType.FIT_XY);
+  
+    private final String mStringValue;
+    private final ScaleType mScaleType;
+  
+    ImageResizeMode(String stringValue, ScaleType scaleType) {
+      mStringValue = stringValue;
+      mScaleType = scaleType;
+    }
+  
+    public String getStringValue() {
+      return mStringValue;
+    }
+  
+    public ScaleType getScaleType() {
+      return mScaleType;
+    }
+  
+    public static ImageResizeMode fromStringValue(String value) {
+      for (ImageResizeMode resizeMode : ImageResizeMode.values()) {
+        if (resizeMode.getStringValue().equals(value)) {
+          return resizeMode;
+        }
+      }
+      return ImageResizeMode.UNKNOWN;
+    }
+  }
```

Comment:
This file is unformatted (or formatted wrongly), it has a 4-space indentation (instead of 2-space) and doesn't have a newline at the end of file. I think the AS settings may be git-tracked only for root Android project. 😕

Would you mind reformatting this file? 🙂

Reply:
Whoops 😅 Reformatted it, thanks! 👍